roles/lib_openshift/src/lib/rolebinding.py

```diff
@@ -0,0 +1,276 @@
+# pylint: skip-file
```

Comment:
Is there a standalone module for rolebinding? Should it be added as a pre-req to this PR rather than just including part of it?